packages/fsapp/src/components/screenWrapper.web.tsx

```diff
@@ -7,13 +7,15 @@ import qs from 'qs';
 import pushRoute from '../lib/push-route';
 import { AppConfigType, DrawerConfig } from '../types';
 
-const styles = StyleSheet.create({
+const StyleSheetCreate: any = StyleSheet.create;
```

Comment:
I'm not sure why you made a separate variable for this, especially because it's only called once.

Reply:
Ah, makes sense. I usually just do a ts-ignore for the specific line, because this shuts off any type checking for the entire object, but this should be fine.